agent.py

```diff
@@ -602,7 +602,8 @@ def parent_func():
                         time.sleep(1)
                         cs = Collector.run_single_check(check, verbose=True)
                         print CollectorStatus.render_check_status(cs)
-
+                    else:
+                        print "Check has run only once, if some metrics are missing you can try again with check_rate appended at the end to see any other metrics if available."
```

Comment:
nit: maybe change to `if some metrics are missing you can run the command again with the 'check_rate' argument appended at the end to see any other metrics if available.`

up to you though 👍

Reply:
I really just copied and pasted from the line we used for Agent 6 to keep things in sync :)

Reply:
mostly I'd add that `check_rate` is an argument, so something like `try again with the 'check_rate' argument appended at the end ` would work for me :)

Reply:
Ahh good point. I changed the wording a bit. Let me know what you think